schemas/sds/1.3/source-data-stream-1.3.sch.original

```diff
@@ -0,0 +1,329 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is the original file from NIST. Do we want to keep this file or should it be removed? This file isn't used at the moment by OpenSCAP. But it's good for a reference.

Reply:
There is no harm in having it in the repo.